clang-format-check/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Clang-format Formatting Check'
+description: 'Waterloo Rocketry clang-format formatting check workflow'
+inputs:
+  clang-format-config-path:
+    description: 'Path to clang-format config file'
+    default: 'rocketlib/.clang-format'
+  c-source-files:
+    description: 'C source code files need to be checked'
```

Comment:
what format does this param require? space-separated list..? that could be specified in the descripion